app/installer/README.md

```diff
@@ -0,0 +1,194 @@
+# marmelab-boilerplate [![Build Status](https://travis-ci.org/marmelab/javascript-boilerplate.svg?branch=master)](https://travis-ci.org/marmelab/javascript-boilerplate)
```

Comment:
IMO a link to original README is enough as you don't need to maintain it yourself.